docs/src/vertical/sbm.md

```diff
@@ -111,31 +111,61 @@ original Topog\_SBM vertical transfer formulation, by specifying in the TOML fil
-ratio between ``U`` and ``S_{d}``: 
+The transfer of water from the ``U`` [mm] store to the ``S`` [mm] store (``st`` [mm
+t``^{-1}``]) is in that case controlled by the saturated hydraulic conductivity ``K_{sat}``
+[mm t``^{-1}``] at depth ``z_{i}`` [mm] and the ratio between ``U`` [mm] and ``S_{d}``
+[mm]: 
 
 ```math
     st=K_{\mathit{sat}}\frac{U_{s}}{S_{d}}
 ```
-Saturated conductivity (``K_{sat}``) declines with soil depth (``z``) in the model according
-to: 
+Saturated conductivity (``K_{sat}`` [mm t``^{-1}``]) declines with soil depth (``z`` [mm])
+in the model according to: 
 
 ```math
     K_{sat}=K_{0}e^{(-fz)}
 ```
-where ``K_{0}`` is the saturated conductivity at the soil surface and ``f`` is a scaling
-parameter [mm``^{-1}``].
+where ``K_{0}`` [mm t``^{-1}``] is the saturated conductivity at the soil surface and ``f``
+is a scaling parameter [mm``^{-1}``].
+
+The plot below shows the relation between soil depth ``z`` and saturated hydraulic
+conductivity ``K_{sat}`` for different values of ``f``.
+
+```@setup plot
+    using Printf
+    using CairoMakie
+```
+
+```@example plot
+    let                                                                                     # hide
+        fig = Figure(resolution = (800, 400))                                               # hide
+        ax = Axis(fig[1, 1], xlabel = "Kₛₐₜ [mm/day]", ylabel = "-z [mm]")                  # hide
+
+        z = 0:5.0:1000                                                                      # hide
+        ksat = 100.0                                                                        # hide
+        f = 0.6 ./ collect(50:150.0:800)                                                    # hide
+
+        for fi in f                                                                         # hide
+            lines!(ax, ksat .* exp.(-fi .* z), -z, label = @sprintf("f = %.2e", fi))        # hide
+        end                                                                                 # hide
+
+        Legend(fig[1, 2], ax, "f")                                                          # hide
+        fig                                                                                 # hide
+    end                                                                                     # hide
+```
 
 ## Transpiration and soil evaporation
 The potential evaporation left over after interception and open water evaporation (rivers
 and water bodies) is split in potential soil evaporation and potential transpiration based
 on the canopy gap fraction (assumed to be identical to the amount of bare soil).
 
-For the case of one single soil layer, soil evaporation is scaled according to:
+For the case of one single soil layer (the SBM soil column is not split-up into different
+layers), soil evaporation [mm] is scaled according to, as shown by the following code block
+(`i` refers to the Vector position):
```

Comment:
what do you mean exactly by the vector position?

Reply:
it is the index of the Vector.

Reply:
ok, you mean the index in space over the grid?

Reply:
yes, internally we work with 1D arrays (based on active cells), so it is the index of the 1D array. feel free to change it, if it is not clear!